### PR TITLE
fzf-make: init at 0.6.0

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Description of changes

Add fzf-make, a fuzzy finder for Makefile

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/shells/fzf-make/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ lib
+, bat
+, rustPlatform
+, fetchFromGitHub
+}:
```
Comment (AndersonTorres) [inline pkgs/shells/fzf-make/default.nix:5]:
```suggestion
}:

```

File: pkgs/shells/fzf-make/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+rustPlatform.buildRustPackage rec {
+  pname = "fzf-make";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "kyu08";
+    repo = pname;
```
Comment (AndersonTorres) [inline pkgs/shells/fzf-make/default.nix:12]:
```suggestion
    repo = "fzf-make";
```

File: pkgs/shells/fzf-make/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    rev = "v${version}";
+    hash = "sha256-Rpz10sHu/pzy/WX1yj0N78gVaw0fO8D92TSdtkmL/MU=";
+  };
+
+  cargoSha256 = "sha256-vy1LJciZMwMo0gxoEtfgESYnUPRhbpnyvciRZHk1Oao=";
```
Comment (AndersonTorres) [inline pkgs/shells/fzf-make/default.nix:17]:
```suggestion
  cargoHash = "sha256-vy1LJciZMwMo0gxoEtfgESYnUPRhbpnyvciRZHk1Oao=";
```

File: pkgs/shells/fzf-make/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  buildInputs = [ bat ];
+
+  meta = with lib; {
+    description = "Fuzzy finder for Makefile";
+    homepage = "https://github.com/kyu08/fzf-make";
+    license = [ licenses.mit ];
+    maintainers = [ maintainers.sigmanificient ];
+    platforms = platforms.unix;
+  };
```
Comment (AndersonTorres) [inline pkgs/shells/fzf-make/default.nix:26]: Don't forget `meta.mainProgram`.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/shells/fzf-make/default.nix (hunk 2 above)
Comment (Sigmanificient) [reply pkgs/shells/fzf-make/default.nix:12]: not sure to understand this one?

**Comment by winterqt** [comment]
(@AndersonTorres Why keep both PRs open, is there something I'm missing here?)

**Comment by Sigmanificient** [comment]
> (@AndersonTorres Why keep both PRs open, is there something I'm missing here?)

Shall we made one pr with both maintainers? Like Co-Authored commits

**Comment by figsoda** [comment]
duplicate of #250014

**Comment by figsoda** [comment]
> Shall we made one pr with both maintainers? Like Co-Authored commit

added you

**Comment by Sigmanificient** [comment]
closed due to duplicate

**Pull request closed without merging**